### PR TITLE
Hello-rails

### PR DESCRIPTION
**Pull request by Rachelwebdev**
In this Milestone, I:

- Set up a new rails project with postgres database
- Created greetings table and ran migrations
- Generated Api  controllers
- Fixed linter errors

### CODE REVIEW

**Comment by Rachelwebdev** [comment]
> In this Milestone, I:

> - Set up a new rails project with postgres database
- Created greetings table and ran migrations
- Generated Api  controllers
- Fixed linter errors

File: README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+## Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Ismailco) [inline README.md]: - [ ] The LICENSE link is broken, kindly consider adding a license to your project and fix this link so your README is more professional

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Rachelwebdev, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Clean code
✔️ Follow requirements 
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rustam-tolipov** [comment]
Hi @Rachelwebdev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 👍 
4. Good API 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**